src/components/WebexActivityStream/TimeRuler.scss

```diff
@@ -1,85 +1,5 @@
-// For Storybook, so that activity stream loads content properly
```

Comment:
None of these styles had to do with the time ruler, so I moved them to `src/components/WebexActivityStream/WebexActivityStream.scss`